app/app/pages/Terms/Terms.tsx

```diff
@@ -1,12 +1,43 @@
 import * as React from 'react'
 import { Link } from 'react-router-dom'
 import { NAV_PROVIDER_SETTINGS } from '../../../provider/provider.links'
+import injectSheet from 'react-jss'
+import trans from '../../../trans'
+const styles = require('./Terms.css');
 
 const Terms = () => (
-  <div>
-    <p>Terms and conditions</p>
-    <Link to={NAV_PROVIDER_SETTINGS}>Continue</Link>
+  <div className={styles.appTermsCover}>
+    <div className={styles.appTermsListCover}>
+      <h2>{trans('app.onboarding.terms.title')}</h2>
+      <div className={styles.termsItem}>
+        <h3>{trans('app.onboarding.terms.title.1')}</h3>
+        <p className={styles.termsItemText}>The following Terms and Conditions (“T&amp;C”) govern the use of the Mysterium open...<a>{trans('app.onboarding.more.btn')}</a></p>
+        <p className={styles.date}>Last updated 2018-12-11</p>
+      </div>
+      <div className={styles.termsItem}>
+        <h3>{trans('app.onboarding.terms.title.1')}</h3>
+        <p className={styles.termsItemText}>The following Terms and Conditions (“T&amp;C”) govern the use of the Mysterium open...<a>{trans('app.onboarding.more.btn')}</a></p>
+        <p className={styles.date}>Last updated 2018-12-11</p>
+      </div>
+      <div className={styles.termsItem}>
+        <h3>{trans('app.onboarding.terms.title.1')}</h3>
+        <p className={styles.termsItemText}>The following Terms and Conditions (“T&amp;C”) govern the use of the Mysterium open...<a>{trans('app.onboarding.more.btn')}</a></p>
+        <p className={styles.date}>Last updated 2018-12-11</p>
+      </div>
+    </div>
+    <div className={styles.bottomBar}>
+      <div className={styles.barContent}>
+        <div>
+          <span className={styles.catMysterium}/>
+          <p className={styles.termsAgreement}>
+            <input type="checkbox" />
+            <span>I agree to all Terms of Serivce</span>
```

Comment:
Clicking on the label should trigger the checkbox as well. One way to achieve it is to nest the input under the label (see below). Also, please use proper HTML elements when needed. In this case, it's not a plain span, but a label. 

```html
<label><input type="checkbox" name="checkbox" value="value">Text</label>
```